### PR TITLE
ISSUE-242 State change implementation

### PR DESCRIPTION
**Pull request by satishwaghela**
UI Implementation for schema version state change.

### CODE REVIEW

**Comment by shahsank3t** [comment]
@satishd Can you please try this PR and provide feedback if any for the UI. Thanks in advance.

**Comment by satishd** [comment]
This UI looks functional for basic state transformations. 
We have complete state trail for each version in the backend which we may want to expose to user. We may also want to show state flow with graphical state flow diagram so that users can easily understand the current state flow of schema registry. It may be useful for users when SR is started with customized state flow. I will raise Github issues for these after finalizing on how we want to expose state transition in UI.

File: webservice/src/main/resources/app/scripts/containers/Registry-Services/SchemaRegistry/SchemaRegistryContainer.jsx [hunk 1]
```diff
@@ -63,6 +65,60 @@ CodeMirror.registerHelper("lint", "json", function(text) {
...
+    this.setState({edit: true}, () => {});
+  }
+  render(){
+    const {edit} = this.state;
+    const {StateMachine, version, showEditBtn} = this.props;
+    const transitions = StateMachine.getTransitionStateOptions(version.stateId);
+    const currentState = StateMachine.getStateById(version.stateId).name;
+    let comp;
+    if(edit){
+      comp = <div style={{"margin-top": "5px"}}>
```
Comment (shahsank3t) [inline webservice/src/main/resources/app/scripts/containers/Registry-Services/SchemaRegistry/SchemaRegistryContainer.jsx]: nit: this should be "marginTop"

**Comment by shahsank3t** [comment]
@satishwaghela once the comment is addressed, +1 from me

**Pull request merged**